### PR TITLE
json-tests moved to tests/ directory

### PR DESCRIPTION
**Pull request by debris**
moved integration json-tests to `ethcore/tests` directory

### CODE REVIEW

**Comment by 5chdn** [comment]
evmbin does not compile.

**Comment by debris** [comment]
I didn't know that `evmbin` depends on them. @tomusdrw what do you think about moving these tests into their own crate?

**Comment by tomusdrw** [comment]
@debris Fine for me. There reason for dependency is that for cross-client testing there is an option to run json test from a command-line. We could split the part that runs the tests from the tests themselves.

**Comment by sorpaas** [comment]
And we'll also need to update this file: `scripts/evm_jsontests_bench.sh`. Also contains reference to jsontests path.

File: ethcore/src/lib.rs [hunk 1]
```diff
@@ -170,15 +171,12 @@ mod cache_manager;
...
 mod account_db;
 mod builtin;
-mod externalities;
 mod blockchain;
 mod factory;
 mod tx_filter;
 
 #[cfg(test)]
 mod tests;
-#[cfg(feature = "json-tests")]
```
Comment (sorpaas) [inline ethcore/src/lib.rs]: This feature probably needs to be removed from `ethcore/Cargo.toml`.

**Comment by 5chdn** [comment]
@debris are you still working on this?

**Comment by debris** [comment]
I'll try coming back to this pr later this week

**Comment by debris** [comment]
If I do not, it can be closed

**Comment by 5chdn** [comment]
Ok

**Pull request closed without merging**